poetry.lock

```diff
@@ -206,7 +206,7 @@ python-versions = "*"
 version = "1.3.1"
 
 [[package]]
-category = "dev"
+category = "main"
```

Comment:
I don't know why but Poetry keeps changing this to be a main dependency but it's a dev dependency. I raised an issue on the Poetry repo some weeks ago but I've not heard back.

Reply:
I'll change this manually to `dev` again (assuming there's no checksum on the lock file preventing manual changes).

Reply:
no it should be fine to change manually. Just not ideal.